src/main/java/org/spdx/tools/SpdxToolsHelper.java

```diff
@@ -81,7 +82,8 @@ public static ISerializableModelStore fileTypeToStore(SerFileType fileType) thro
 		case XLSX: return new SpreadsheetStore(new InMemSpdxStore(), SpreadsheetFormatType.XLSX);
 		case XML: return new MultiFormatStore(new InMemSpdxStore(), Format.XML, Verbose.COMPACT);
 		case YAML: return new MultiFormatStore(new InMemSpdxStore(), Format.YAML, Verbose.COMPACT);
-		default: throw new InvalidSPDXAnalysisException("Unsupporte file type: "+fileType+".  Check back later.");
+		case YML: return new MultiFormatStore(new InMemSpdxStore(), Format.YAML, Verbose.COMPACT);
```

Comment:
You no longer need this case statement.  The additional file type will get taken care of by the line `SerFileType retval = EXT_TO_FILETYPE.get(ext);` below..